docs/principles-for-package-repository-security.md

```diff
@@ -0,0 +1,105 @@
+# Principles for Package Repository Security
+
+Authors: [Jack Cable](https://github.com/cablej), [Zach Steindler](https://github.com/steiza)
+Last updated: Dec 2023
+
+## Background
+
+The Securing Software Repositories Working Group (WG) of the OpenSSF is putting together a taxonomy of package repositories and a set of principles for security capabilities. This is intended to be an opinionated document, offering a set of best practices that package repositories should strive to adhere to.
```

Comment:
```suggestion
The Securing Software Repositories Working Group (WG) of the OpenSSF has identified a taxonomy of package repositories and a set of principles for their security capabilities. This is intended to be an opinionated document, offering a set of best practices that package repositories should strive to adhere to.
```
Seems weird that this would be in future tense, assuming this describes this document, correct?